src/docker-compose.yml

```diff
@@ -11,3 +11,8 @@ services:
     build:
       context: .
       dockerfile: Arcus.Messaging.Tests.Workers.ServiceBus.Topic/Dockerfile
+  arcus.messaging.tests.runtimes.azurefunction.servicebus.queue:
```

Comment:
We don't need this since Docker Compose doesn't like Azure Functions

https://github.com/microsoft/DockerTools/issues/124